### DISCUSSION TITLE
this project works with exyon

### DISCUSSION THREAD

**Discussion by jgsjnr** [Samsung/ONE#12756] [General]
Hi, this project works with exynos 2400? This is a substitute to Samsung Neural SDK?

**Comment by hseok-oh** [comment]
It is independent project with Samsung Neural SDK. 
cc/ @lemmaa

**Reply by EwoutH** [reply]
Do you expect to add Exynos NPU support at some point in this project? It seems that a huge increase in performance is possible from running on Exynos NPUs, since they generally have quite powerful NPUs compared to their GPUs.
